wormhole-connect/README.md

```diff
@@ -82,3 +82,36 @@ class WormholeConnect extends React.Component {
   }
 }
 ```
+
+## Togle Hamburguer Menu
+
+by setting `showHamburgerMenu` to **false**, you can toggle hamburguer menu off
+and that makes the links to be placed at bottom
+
+### Add extra menu entry
+
+by setting `showHumburgerMenu` to **false**, you can use `menu` array to add
+extra links
```

Comment:
By setting the `showHumburgerMenu` option to **false**, you can use the `menu` array to add extra links.

Reply:
thanks, updated